src/components/search-by-message/SearchByMessage.styles.tsx

```diff
@@ -0,0 +1,14 @@
+export const styles = {
+  container: {
+    width: '100%',
+    maxHeight: '56px',
+    display: 'flex',
+    justifyContent: 'space-between',
+    padding: '8px 0px',
+    backgroundColor: 'rgba(252, 252, 252, 1)',
```

Comment:
We have our colors in the `app.pallete.ts` file. If you need to use this particular color, try to add it there and use it for example like this:
`backgroundColor: 'basic.white'`
that would be nice

Reply:
Do you need this color, because as I see it should be white, like header in chat

Reply:
done

> We have our colors in the `app.pallete.ts` file. If you need to use this particular color, try to add it there and use it for example like this: `backgroundColor: 'basic.white'` that would be nice

done

Reply:
> Do you need this color, because as I see it should be white, like header in chat

i think it`s needed